src/game/default/g_item.c

```diff
@@ -882,7 +882,7 @@ void G_PrecacheItem(const g_item_t *it) {
 		else if (g_str_has_suffix(data, ".wav") || g_str_has_suffix(data, ".ogg"))
 			gi.SoundIndex(data);
 		else if (g_str_has_suffix(data, ".tga") || g_str_has_suffix(data, ".png")
-				|| g_str_has_suffix(data, ".jpg") || g_str_has_suffix(data, ".pcx"))
```

Comment:
Assets might still reference .jpg textures, and SDL_image will still load .jpg just fine for us.